tools/options.py

```diff
@@ -121,3 +121,12 @@ def extract_profile(parser, options, toolchain, fallback="develop"):
                                 " supported by profile {}").format(toolchain,
                                                                    filename))
     return profile
+
+def not_disabled(parser, mcu):
```

Comment:
It's named that way so that I can use assert in front of it. Even if asserting "not_disabled" is not actually what happens, it documents the intent well. The intent being Fail now, hard or keep going.